modules/promotion/src/Element/CouponRedemptionForm.php

```diff
@@ -45,6 +50,8 @@ public function getInfo() {
       // The coupon ID.
       '#default_value' => NULL,
       '#order_id' => NULL,
+      '#display_actions' => TRUE,
```

Comment:
I don't know why this was put there. Need to review issue comments. Not sure why you would embed the element but not display actions? To list coupons?